time time
---
bool started
bool ended
duration elapsedTime
geometry_msgs/PointStamped point
